build/parsePrep.c

```diff
@@ -273,6 +273,15 @@ static int doSetupMacro(rpmSpec spec, const char *line)
 	free(buf);
     }
 
+    /* mkdir for dynamic specparts */
+    buf = rpmExpand("%{__mkdir} SPECPARTS", NULL);
+    appendBuf(spec, buf, 1);
+    free(buf);
+
+    buf = rpmGenPath("%{u2p:%{_builddir}}", "%{buildsubdir}", "SPECPARTS");
+    rpmPushMacro(spec->macros, "specpartsdir", NULL, buf, RMIL_SPEC);
```

Comment:
This defines the `specpartsdir` macro for use in a specpart file, however the directory name `SPECPARTS` is still hardcoded. Is that what we want?

Reply:
Indeed, I was thinking about a macro for SPECPARTS specifically, and drop any references to the explicit value from both the code *and documentation*.

Reply:
The actual dir name is now removed from the docs but remains hard coded.